ocs_ci/krkn_chaos/background_cluster_operations.py

```
@@ -156,12 +160,88 @@ def __init__(
+        return 1
+
+    def _is_cephx_keyrotation_supported(self) -> bool:
+        if config.DEPLOYMENT.get("external_mode"):
+            return False
+        try:
+            from ocs_ci.utility import version
+
+            return (
+                version.get_semantic_ocs_version_from_config() >= version.VERSION_4_19
+            )
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)